sdk/identity/identity/src/credentials/interactiveBrowserCredentialOptions.ts

```diff
@@ -42,6 +42,11 @@ export interface InteractiveBrowserCredentialNodeOptions
    * Setting this option skips the account selection prompt and immediately attempts to login with the specified account.
    */
   loginHint?: string;
+
+  /**
+   * Enables MSA Passthrough to interactive browser authentication.
+   */
+  enableMsaPassthrough?: boolean;
```

Comment:
Interface that derives from this inside the broker package. which accepts `enableMsaPassthrough`, `parentWindowHandle` and `useBroker` (need to specify this explicitly, this regardless if it's loaded)